.github/workflows/plone-package.yml

```diff
@@ -0,0 +1,50 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Plone package
+
+on:
+    push:
+        branches: [main]
```

Comment:
remove branches to allow any branches? tests did not run yet.

Reply:
This is meant for the target branch, not the source branch. The build failed, but I'm unsure why. The tests can be run manually however.